draft/ZEP0003.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title: ZEP0003
+description: Variable chunk sizes
+parent: draft ZEPs
+nav_order: 3
+---
+
+# ZEP 3 — Variable chunking
+
+Authors:
+* Martin Durant ([@martindurant](https://github.com/martindurant)), Anaconda, Inc.
```

Comment:
```suggestion
* Martin Durant ([@martindurant](https://github.com/martindurant)), Anaconda, Inc.
* Isaac Virshup ([@ivirshup](https://github.com/martindurant)), Helmholtz Munich
```

Any objection to adding me here? I think a fair amount of the text from my hackmd draft has made it here.